crates/sncast/tests/e2e/script/init.rs

```diff
@@ -0,0 +1,135 @@
+use camino::Utf8PathBuf;
+use indoc::{formatdoc, indoc};
+use snapbox::cmd::{cargo_bin, Command};
+use sncast::helpers::constants::SCRIPTS_DIR;
+use sncast::helpers::scarb_utils::get_cairo_version;
+use tempfile::TempDir;
+
+#[test]
+fn test_script_init_files_created() {
```

Comment:
```suggestion
fn test_script_happy_case() {
```

nitpick, to conform to conventions already present in repo